code/_globalvars/traits.dm

```diff
@@ -108,7 +108,10 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_DWARF" = TRAIT_DWARF,
 		"TRAIT_GIANT" = TRAIT_GIANT,
 		"TRAIT_SILENT_FOOTSTEPS" = TRAIT_SILENT_FOOTSTEPS,
-		"TRAIT_NICE_SHOT" = TRAIT_NICE_SHOT
+		"TRAIT_NICE_SHOT" = TRAIT_NICE_SHOT,
+		"TRAIT_NOBREAK" = TRAIT_NOBREAK,			// Wasp edit - Whitesands
+		"TRAIT_ALLBREAK" = TRAIT_ALLBREAK,			// Wasp edit - Whitesands
+		"TRAIT_NOLIMBDISABLE" = TRAIT_NOLIMBDISABLE		// Wasp edit - Whitesands
```

Comment:
This is already a trait. https://github.com/WaspStation-Bot/WaspStation-1.0/blob/27d7ddeedee1d8233cda97b6ce0e130caf42751e/code/_globalvars/traits.dm#L54
https://github.com/WaspStation-Bot/WaspStation-1.0/blob/27d7ddeedee1d8233cda97b6ce0e130caf42751e/code/__DEFINES/traits.dm#L129